modules/cluster-logging-deploy-cli.adoc

```diff
@@ -4,9 +4,9 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-deploy-cli_{context}"]
-= Installing the {logging-title} using the CLI
+= Installing the {logging} using the CLI
 
-You can use the {product-title} CLI to install the OpenShift Elasticsearch and Red Hat OpenShift Logging Operators.
+You can use the {product-title} CLI to install the {es-op} and the {clo}.
```

Comment:
```suggestion
You can use the {oc-first} to install the {es-op} and the {clo}.
```
Since we use `oc` after the first mention, I'd use the attribute here.